plugin/trino-ml/src/main/java/io/trino/plugin/ml/SvmClassifier.java

```diff
@@ -50,7 +51,7 @@ public static SvmClassifier deserialize(byte[] modelData)
     {
         // TODO do something with the hyperparameters
         try {
-            svm_model model = svm.svm_load_model(new BufferedReader(new InputStreamReader(new ByteArrayInputStream(modelData))));
+            svm_model model = svm.svm_load_model(new BufferedReader(new InputStreamReader(new ByteArrayInputStream(modelData), UTF_8)));
```

Comment:
what does this read? where does `modelData` come from?

Reply:
Hm. Looks like nobody calls this method?

Reply:
Ah, it's called dynamically in `io.trino.plugin.ml.ModelUtils#deserialize(io.airlift.slice.Slice)`. The `byte[] data` comes from the`Slice`. I have no idea what are the contents of the slice - some representation of the ML model, it appears, and it doesn't look like a text 🤔

Reply:
ouch. use UTF-8 and add a comment that this may or may not be correct choice